src/Symfony/Component/HttpKernel/Config/EnvParametersResource.php

```diff
@@ -82,6 +82,12 @@ public function unserialize($serialized)
         $this->variables = $unserialized['variables'];
     }
 
+    /**
+     *  Find environment variables starting with
+     *  the $prefix property.
+     *
+     * @return array
+     */
```

Comment:
I would say that this is unneeded. We don't add phpdoc for private method, and here, it looks clear to me without adding anything.

Reply:
Thank you for reviewing @fabpot, and for your feedback. I'll close the PR :)